tests/testthat/test-epidist.R

```diff
@@ -17,9 +17,9 @@ test_that("epidist works with valid input", {
 
 test_that("epidist prob functions work", {
   marburg_dist <- epidist(pathogen = "marburg", delay_dist = "incubation")
-  expect_equal(marburg_dist$pmf(5), 0.19445393887)
-  expect_equal(marburg_dist$pdf(5), 0.0822904644827)
-  expect_equal(marburg_dist$cdf(5), 0.0305977451594)
+  expect_equal(marburg_dist$pmf(5), 0.000641548322219965)
+  expect_equal(marburg_dist$pdf(5), 0.00133380313883437)
+  expect_equal(marburg_dist$cdf(5), 0.999218672257759)
```

Comment:
Why such a big change in these values?

Reply:
The source of the marburg data was changed from Martini 1973 (10.1136/pgmj.49.574.542) to Pavlin 2014 (10.1186/1756-0500-7-906). I agree the change is drastic, so something to check on. I'll link this to a new issue.